docs/source/developing/writing-docs.adoc

```diff
@@ -10,29 +10,93 @@ toc::[]
 [[writing-docs-overview]]
 == Overview
 
-This section contains information about contributing to the Minishift documentation.
-
-[[section-building-minishift-docs]]
-== Building Minishift documentation
-
 Minishift documentation is located in the `docs` sub-folder.
 The documentation is a mix of generated https://en.wikipedia.org/wiki/Markdown[Markdown] files and manually maintained https://en.wikipedia.org/wiki/AsciiDoc[Asciidoc] files.
 
-If you are adding a new file to the doumentation please update the `_topic_map.yml`  in the root of `docs/source` sub-folder.
+[[contribute-to-docs]]
+== Contributing to the documentation
+
+Minishift is an open-source project and we welcome contributions. Similar to code
+contributions, if you want to add or edit Minishift documentation, you can create an
+issue in our link:https://github.com/minishift/minishift/issues[Github issue tracker] and
+submit a pull request with the changes.
+
+[[docs-conventions-guidelines]]
+== Conventions and guidelines
+
+Minishift documentation is authored in the AsciiDoc markup format. Since Minishift is closely related to the OpenShift Origin
+project, we follow the link:https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc[OpenShift Documentation Guidelines]
+for tagging, formatting, and structure wherever possible.
+
+The documentation source structure follows a modular format, where each file is called
+a _topic_ and each topic is stored under a sub-folder based on the relevant category, such as
+getting started, using Minishift, or command reference.
+
+TIP: You can check out the link:https://github.com/minishift/minishift/blob/master/docs/source/developing/writing-docs.adoc[raw file]
+of this topic or other existing Minishift topics to see examples of how the information is structured and which tags to use.
+
+[[common-conventions]]
+=== Commonly-used conventions
+
+- Each file must contain a mandatory header section with metadata that is required for publication.
+The only exception to this is the command reference topics, which are automatically generated and
+do not require this header. See the link:https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#topic-metadata[topic metadata]
+section in the OpenShift documentation guidelines for more information.
+
+- Whenever we link to internal documentation topics, you should use relative links that can
```

Comment:
I mentioned the relative links because some people tend to copy and paste the published URL of a certain topic instead of the relative link to the raw file.